bin/alluxio

```diff
@@ -247,6 +185,7 @@ function copyDir {
 }
 
 PARAMETER=""
+ALLUXIO_SHELL_JAVA_OPTS=""
 
 case ${COMMAND} in
```

Comment:
basically the only non-function control flow in this script should be:

`main "$@"`